boards/arm/stm32f429i_disc1/doc/stm32f429i_disc1.rst

```diff
@@ -0,0 +1,203 @@
+
+.. image:: img/en.stm32f429i_disc1.png
+     :width: 350px
+     :align: center
+     :height: 631px
+     :alt: STM32F429I-DISC1
+
+More information about the board can be found at the `STM32F429IDISCOVERY website`_.
+
+Hardware
+********
+
+STM32F429I-DISC1 Discovery kit provides the following hardware components:
+
+- STM32F429ZIT6 in LQFP144 package
+- ARM®32-bit Cortex®-M4 CPU with FPU
+- 180 MHz max CPU frequency
+- VDD from 1.8 V to 3.6 V
+- 2 MB Flash
+- 256+4 KB SRAM including 64-Kbyte of core coupled memory
+- GPIO with external interrupt capability
+- 3x12-bit ADC with 24 channels
+- 2x12-bit D/A converters
+- RTC
+- Advanced-control Timer
+- General Purpose Timers (17)
+- Watchdog Timers (2)
+- USART/UART (4/4)
+- I2C (3)
+- SPI (6)
+- SDIO
+- 2xCAN
+- USB 2.0 OTG FS with on-chip PHY
+- USB 2.0 OTG HS/FS with dedicated DMA, on-chip full-speed PHY and ULPI
+- 10/100 Ethernet MAC with dedicated DMA
+- 8- to 14-bit parallel camera
+- CRC calculation unit
+- True random number generator
+- DMA Controller
+
+More information about STM32F429ZI can be found here:
+       - `STM32F429ZI on www.st.com`_
+       - `STM32F429 reference manual`_
+
+Supported Features
+==================
+
+The Zephyr stm32f429i_disc1 board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on Zephyr porting.
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/stm32f429i_disc1/stm32f429i_disc1_defconfig``
+
+
+Pin Mapping
+===========
+
+STM32F429I-DISC1 Discovry kit has 8 GPIO controllers. These controllers are responsible for pin muxing,
+input/output, pull-up, etc.
+
+For mode details please refer to `STM32F429-DISC1 board User Manual`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+- UART_1_TX : PA9
+- UART_1_RX : PA10
+- UART_2_TX : PA2
+- UART_2_RX : PA3
+- USER_PB : PA0
+- LD3 : PG13
+- LD4 : PG12
+
+System Clock
+============
+
+STM32F429I-DISC1 System Clock could be driven by internal or external oscillator,
```

Comment:
Could use some articles ("the", "an"):

     The STM32F429I-DISC1 system clock can be driven by an internal or external
     oscillator, as well as by the main PLL clock. By default the system clock is
     driven by the PLL clock at 168MHz, driven by an 8MHz high speed external
     clock.